content/docs/static-analysis/semgrep/20-ci.md

```diff
@@ -35,6 +35,60 @@ The following are key points to note:
   by separating them with a space.
 - By default, the CI process fails if findings are detected but passes if internal errors occur. For more information, see
 [Passing or failing the CI job](https://semgrep.dev/docs/semgrep-ci/running-semgrep-ci-without-semgrep-app/#passing-or-failing-the-ci-job).
+- See the example job that uploads findings to [GitHub Advanced Security Dashboard](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs#:~:text=Alternate%20job%20that%20uploads%20findings%20to%20GitHub%20Advanced%20Security%20Dashboard).
+
+#### Adding custom Semgrep rules to CI/CD
+
+When you want to use your own custom rules in addition to the standard rulesets (such as `p/default` or `p/javascript`)
+passed to the `SEMGREP_RULES`, follow the steps below:
+
+1. If your custom Semgrep rules directory **is in the same** repository as the scanned code,
+       just pass the directory path in the `SEMGREP_RULES` variable:
+      (e.g., `SEMGREP RULES: p/default custom-semgrep-rules-dir/`)
+
+2. If your custom Semgrep rules **are in another private repository**, do the following:
+
+    a. [Generate an access token for the repository with Semgrep rules](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+    Remember to select the least scopes necessary.
+
+    b. [Add the generated access token](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions)
+    to the repository where the workflow is run
+
+    c. Add the `actions/checkout` step in a job after the main source code checkout with:
+      - The `repository` name
+      - Personal access `token` (PAT) used to fetch the repository
+      - Relative `path` to place the repository
+
+    d. Pass the path to the directory with custom Semgrep rules in the `SEMGREP_RULES` environment variable
+
+    {{< hint info >}}
+If your repository with custom rules is publicly available, just omit the steps where you create the PAT and do not pass
+the `token` in the checkout step.
+    {{< /hint >}}
+
+    For example:
+
+    ```yaml
+    # Set up an environment variable with the name of the private repository with custom Semgrep rules
+    env:
+      SEMGREP_PRIVATE_RULES_REPO: semgrep-private-rules
```

Comment:
Does it need to be under the same org? Or can you do other-user/semgrep-private-rules?

Reply:
Nevermind, I see below